pkgs/development/compilers/llvm/11/libcxxabi/default.nix

```diff
@@ -1,4 +1,6 @@
 { lib, stdenv, llvm_meta, cmake, fetch, libcxx, libunwind, llvm, version
+, standalone ? stdenv.hostPlatform.useLLVM or false
+, withLibunwind ? !stdenv.isDarwin && !stdenv.isFreeBSD && !stdenv.hostPlatform.isWasm
```

Comment:
I've done this for LLVM 12 and 13 now. Older LLVM posed some challenges so I'd rather skip them. The only place this is in use so far is the Darwin stdenv and we wouldn't use older LLVMs for that anyway, going forward.

I do want to note that this is basically a cargo-cult of the changes for LLVM 7. I did run into this problem [here](https://gist.github.com/toonn/2dfb04d075db4bdaedb0504339a74046#stdenv-bootstrap-stage-3-ends-up-depending-on-llvm-11-libunwind) but I can't find a way to reproduce it. Even the example from the commit making the changes for LLVM 7 (fece3eb2e981b3bd9ea73b34f302ff3aa3d9f81c) doesn't help because I cannot figure out how to generate `<hash>-libc++-<version>`, note the `++` rather than `xx`.

Reply:
> Older LLVM posed some challenges so I'd rather skip them.

That's unfortunate, but the main thing is that it's in the most recent version, so it gets carried forward when 13 is copied to 14, etc, so I think it's okay.

Reply:
Talked to @thefloweringash about this. The reason I couldn't generate a path with `libc++` is that [this changed in a refactor](https://github.com/NixOS/nixpkgs/commit/e0d3c9d0317edf7fd8d99e78066a7cdaf1c5da21)...